meson.build

```diff
@@ -501,6 +501,10 @@ foreach ident : [
                                  #include <unistd.h>'''],
         ['pivot_root',        '''#include <stdlib.h>
                                  #include <unistd.h>'''],     # no known header declares pivot_root
+        ['ioprio_get',        '''#include <stdlib.h>
+                                 #include <unistd.h>'''],     # no known header declares ioprio_get
```

Comment:
smells like something that would get added to `<sched.h>`, if it ever gets added, maybe look there too?